### ISSUE TITLE
The `@byrow` example in the README doesn't work

### ISSUE THREAD

**Issue by xiaodaigh** [JuliaData/DataFramesMeta.jl#164]
```
using DataFrames
using DataFramesMeta
df = DataFrame(A = 1:3, B = [2, 1, 2])
df2 = @byrow df begin
    @newcol colX::Array{Float64}
    @newcol colY::Array{Union{Int,Missing}}
    :colX = :B == 2 ? pi * :A : :B
    if :A > 1
        :colY = :A * :B
    else
        :colY = Missing
    end
end
```

**Comment by pdeffebach** [comment]
Yes, we should

1. Make a patch release with the new `@byrow` naming. It makes sense to do this now that `@cols` in `@byrow` works. 
2. Use actual documentation instead of a readme so that we can have separate documentation for master vs. release branch.

**Comment by bkamins** [comment]
OK

**Comment by pdeffebach** [comment]
Closing this, it was fixed when we added real documentation in #208.
